gtk/src/auto/enums.rs

```diff
@@ -7686,116 +7506,6 @@ impl ToValue for StackTransitionType {
     }
 }
 
-#[deprecated = "Since 3.14"]
```

Comment:
Should maybe update the minimum supported version before this release... 3.14 is from 2014. That's ancient :) Maybe 3.20 would be a good next step, that's 2016 so would still cover Ubuntu 16.04 for example.

@GuillaumeGomez @bilelmoussaoui What do you think? That would be a separate PR of course.

Reply:
Sounds good to me.

Reply:
No opinions on that, but seems like everyone has 3.20 nowadays